recipes/wellmap/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "wellmap" %}
+{% set version = "3.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wellmap-{{ version }}.tar.gz
+  sha256: 1957fec63bdc4bea6af359fa8013135786413826f8e6eb50d50117cc4138e9f0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - wellmap = wellmap.plot:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - flit-core
```

```suggestion
    - flit-core >=3.2,<4
```
Ref: https://github.com/kalekundert/wellmap/blob/cd391b04b3d0f9a3372d0a9e4307f57e1ddea033/pyproject.toml#L2